zipline/data/minute_bars.py

```diff
@@ -549,7 +549,9 @@ def last_date_in_output_for_sid(self, sid):
         with open(sizes_path, mode='r') as f:
             sizes = f.read()
         data = json.loads(sizes)
-        num_days = data['shape'][0] / self._minutes_per_day
+        # use integer division so that the result is an int
+        # for pandas index later https://goo.gl/VAW649
```

Comment:
Ok great. Just one last thing; we should keep this link expanded (so people know it's a safe link to click). You can make the flake8 tests ignore the length of this line by adding `# noqa` line 553. So it would look like:

`# for pandas index later <link> # noqa`

Should've mentioned that before, sorry! (after that it should be good to merge)